services/ui-src/src/components/tables/EntityRow.test.tsx

```diff
@@ -39,9 +39,12 @@ const mockedGetEntityStatus = getEntityStatus as jest.MockedFunction<
   typeof getEntityStatus
 >;
 
-const completeRowComponent = (props: EntityRowProps) => (
+const completeRowComponent = (
+  props: EntityRowProps,
+  context: ReportContextShape
```

Comment:
split context from props so we can remove the `[key: string]: any` from `EntityRowProps`